pkg/grpc/grpc_test.go

```diff
@@ -0,0 +1,17 @@
+package grpc
```

Comment:
```suggestion
package grpc_test
```

Reply:
I'm testing private package function, it needs to be in the same package